guide/popular-topics/miscellaneous-examples.md

```diff
@@ -185,6 +185,73 @@ Here's what the embed from code above would look like:
 
 <p class="tip">If you want to instantly preview what your embed will look like, check out [this site](https://leovoel.github.io/embed-visualizer/)!</p>
 
+### Converting MessageEmbeds
+
+If you ever tried to fetch the embed from an existing message and resend it (maybe in another channel) you will know that this is not easily possible.
+Received embeds are of the class [MessageEmbed](https://discord.js.org/#/docs/main/stable/class/MessageEmbed) while outgoing embeds have to be of the class [RichEmbed](https://discord.js.org/#/docs/main/stable/class/RichEmbed). 
+
+The RichEmbed class does have a constructor taking in embed data, this will however cause conflicts with the circular nature of MessageEmbeds. 
+To solve this issue you can construct a new RichEmbed and apply the correlating values of the received MessageEmbed as demonstrated below. 
+
+The following code sample results in a bot re-posting any and all embeds it reads:
+
+```js
+const Discord = require('discord.js');
+const client = new Discord.Client();
+
+const toRichEmbed = (messageEmbed) => {
```

Comment:
Generally for lengthy functions you use a "fat function" (function toRichEmbed() { }) arrow function are used for much smaller functions or Higher-Order-Functions.

Reply:
While I agree that, personally, I would have used a function keyword here too it is pretty much a false statement to say that it's only used for "smaller functions" or higher-order functions.